src/parameter/parametervirtualwidget.h

```diff
@@ -5,6 +5,8 @@
 #include "ui_ParameterEntryWidget.h"
 #include "parameterobject.h"
 
+enum descLoD {ShowDetails,Inline,HideDetails,DescOnly};
```

Comment:
Prefer `enum class`, to get some extra type safety.
..and Capitalize the first letter for good measure.

Reply:
good idea. It results in just one awkward line, where I have to static_cast comboBoxDetails->currentIndex() - but that´s fine as static_cast is precisely the right indicator for whats going on.